packages/zapp/console/src/components/Executions/Tables/NodeExecutionActions.tsx

```diff
@@ -122,7 +105,26 @@ export const NodeExecutionActions = ({ execution }: NodeExecutionActionsProps):
           <InputsAndOutputsIcon />
         </IconButton>
       </Tooltip>
-      {renderRerunAction()}
+      <Tooltip title={t('rerunTooltip')}>
+        <IconButton onClick={rerunIconOnClick}>
+          <RerunIcon />
+        </IconButton>
+      </Tooltip>
+      <LaunchFormDialog
+        id={id as ResourceIdentifier}
+        initialParameters={initialParameters}
+        showLaunchForm={showLaunchForm}
+        setShowLaunchForm={setShowLaunchForm}
+      />
```

Comment:
I think this should be wrapped in id and initialParameters presence check
```suggestion
      {id && initialParameters && (<Tooltip title={t('rerunTooltip')}>
        <IconButton onClick={rerunIconOnClick}>
          <RerunIcon />
        </IconButton>
      </Tooltip>
      <LaunchFormDialog
        id={id as ResourceIdentifier}
        initialParameters={initialParameters}
        showLaunchForm={showLaunchForm}
        setShowLaunchForm={setShowLaunchForm}
      />
      )}
```